website/docs/d/kms_key_policy.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+subcategory: "Key Management Service"
+layout: "ibm"
+page_title: "IBM : kms-key-policies"
+description: |-
+  Manages an IBM hs-crypto or key-protect key policies.
+---
+
+# ibm\_kms_key
+
+Import the details of existing hs-crypto or key-protect keys policies as a read-only data source. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax. Retreives a list of key policies from the hs-crypto or key-protect instance for the provided key id.
+
+## Example Usage
+
+```terraform
+data "ibm_kms_key_policies" "test" {
+  instance_id = "guid-of-keyprotect-or hs-crypto-instance"
+  key_id = "key-id-of-the-key"
+}
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `instance_id` - (Required, string) The keyprotect instance guid.
+* `key_id` - (Required, string) The id of the key.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+  * `id` - The unique identifier for this key
+  * `key_id` - The key ring id for the key.
+    * `rotation` - The key rotation time interval in months, with a minimum of 1, and a maximum of 12.
```

Comment:
rotation is outer attribute.. Please remove space infront of rotation..

Follow the guide lines of latest docs template
https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/kms_key#policies
Before describing inner attributes of rotation please following line
Nested scheme for rotation:

Similar changes for 
dual_auth_delete